src/main/java/racingcar/step3/Race.java

```diff
@@ -0,0 +1,31 @@
+package racingcar.step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class Race {
+    public List<Car> getCars() {
+        return cars;
+    }
+
+    private List<Car> cars = new ArrayList<>();
+
+    public Race(int numberOfCars) {
+        for (int i = 0; i < numberOfCars; i++) {
+            cars.add(new Car());
+        }
+    }
+
+    public void start(int attempts) {
+        System.out.println("실행 결과");
+        for (int i = 0; i < attempts; i++) {
+            for (Car car : cars) {
+                if (i != 0) {
+                    car.move();
+                }
```

Comment:
```suggestion
                    car.move();
```
if 분기가 없어도 될 것 같은데 이유가 있나요?

Reply:
초기값 출력하려고 했습니다.

Reply:
제 생각과 좀 다른 것 같습니다.

만약 시도 카운트가 3번이라면 3번 움직이는 시도를 해야하는데 실질적으로 최대 2번밖에 움직이지 않겠네요
이런 이유로 if 분기를 제거하는 것이 올바를 것 같습니다.

아무것도 이동되지 않은 초기 출력값(`-`)을 출력하고 싶다면
출력되는 부분에서 +1 하는 부분을 유지하면 될 것 같네요